static/scss/partials/dashboard.scss

```diff
@@ -95,18 +95,11 @@
         box-shadow: var(--dropShadowLight);
```

Comment:
Do we have strong feelings on the mix of `var()` syntax and using Sass variables?

Reply:
Yeah – Goal is to have no `--var/var()` variables.